media/com_associations/js/admin-autoassoc-modal.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+(function() {
+  "use strict";
+
+  Joomla.gatherCheckedBoxes = function(remember) {
+    var checkedBoxes = document.querySelectorAll("td.row-selected input[type='checkbox']");
+    var values = [];
+
+    checkedBoxes.forEach(function(box) {
+      values.push(box.value);
+    });
+
+    window.parent['fillFields'](values, remember);
+    window.parent['closeModal']();
```

Comment:
There is an API method to close the modal:
`window.parent.Joomla.Modal.getCurrent().close()`

Reply:
@dgrammatiko 
This PR is obsolete.
All is now done in new branch new-staging.
https://github.com/joomla-projects/gsoc18_automatic_multilingual_association/pull/6